vision/set_endpoint.rb

```diff
@@ -0,0 +1,39 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+def set_endpoint service_address: nil, image_path: nil
+  # [START vision_set_endpoint]
+  # service_address = "Service address, eg. 'eu-vision.googleapis.com'"
+  # image_path = "Google Cloud Storage URI, eg. 'gs://my-bucket/image.png'"
```

Comment:
This should not be in the region tag for this snippet.

Reply:
in that case the code snippet would be creating a client that passes in a variable `service_address` (which appears to be what the Ruby samples typically do) without hints to the reader what that service address should look like.

@TheRoyalTnetennba in this case is it acceptable to hard code the service address instead of using a variable?

Reply:
I more meant `image_path` should not, it isn't part of what will be on the docs page.

Reply:
updated to be more consistent with the existing reference page: https://cloud.google.com/natural-language/docs/locations#automl_set_endpoint-python

Reply:
I think hardcoding the address is fine given the context of the docs

Reply:
Could you surround `def set_endpoint image_path: nil` with
`# rubocop:disable Naming/AccessorMethodName
# rubocop:enable Naming/AccessorMethodName` so it can find happiness?